lib/generators/scenic/view/view_generator.rb

```diff
@@ -8,6 +8,10 @@ class ViewGenerator < Rails::Generators::NamedBase
       source_root File.expand_path("../templates", __FILE__)
 
       def create_view_definition
+        unless views_path.exist?
```

Comment:
Actually, I would rename the existing one to 'views_directory_path' and create another one called 'create_views_directory'.